### PR TITLE
V raniv/call automation package update

### PR DESCRIPTION
**Pull request by Rani0608**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder.csproj [hunk 1]
```diff
@@ -7,14 +7,10 @@
...
 
   <ItemGroup>
+    <PackageReference Include="Azure.Communication.CallAutomation" Version="1.0.0-alpha.20230414.1" />
     <PackageReference Include="Azure.Communication.Identity" Version="1.2.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
-	<PackageReference Include="Azure.Communication.CallAutomation" Version="1.0.0-beta.1" />
-  </ItemGroup>
-
-  <ItemGroup>
```
Comment (moirf) [inline CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder.csproj:16]: why you removed this, did you check if this code working after publish to azure?

File: CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/appsettings.json [hunk 2]
```diff
@@ -7,15 +7,24 @@
   },
   "CallConfiguration": {
     "ConnectionString": "%Connection String%",
-    "SourcePhoneNumber": "%Source Phone Number%",
+    "SourcePhoneNumber": "%Source PhoneNumber%",
     "TargetPhoneNumber": "%Target or Destination Phone number%",
-    "AppBaseUri": "%App base uri%",
+    //Add Muliple Participant Number suparated by ";" eg: +919300000000;8:acs:5e4ac494-3bb3-4bd9-aeba-018fd0bcac9b_00000017-eb7a-fc77-f6c7-00000000000
+    "AddParticipantNumber": "%Add Muliple Participant Number %",
```
Comment (moirf) [inline CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/appsettings.json:13]: AddParticipantNumber  replace with AddParticipantList

File: CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/appsettings.json [hunk 3]
```diff
@@ -7,15 +7,24 @@
...
     "EventCallBackRoute": "/api/callbacks",
     "AppointmentReminderMenuAudio": "/audio/AppointmentReminderMenu.wav",
     "AppointmentConfirmedAudio": "/audio/AppointmentConfirmedAudio.wav",
     "AppointmentCancelledAudio": "/audio/AppointmentCancelledAudio.wav",
+    "AgentAudio": "/audio/AgentAudio.wav",
+    "AddParticipant": "/audio/AddParticipant.wav",
+    "RemoveParticipant": "/audio/RemoveParticipant.wav",
     "InvalidInputAudio": "/audio/InvalidInputAudio.wav",
-    "TimedoutAudio": "/audio/TimedoutAudio.wav"
+        "TimedoutAudio": "/audio/TimedoutAudio.wav",
```
Comment (moirf) [inline CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/appsettings.json:23]: formatting

File: CallAutomation_SimpleIvr/Program.cs [hunk 4]
```diff
@@ -131,15 +134,47 @@
         }
         if (@event is PlayCompleted)
         {
+            //if (@event.OperationContext == "AgentConnect")
```
Comment (moirf) [inline CallAutomation_SimpleIvr/Program.cs:137]: remove commented

File: CallAutomation_SimpleIvr/Program.cs [hunk 5]
```diff
@@ -131,15 +134,47 @@
...
-                await callConnection.AddParticipantsAsync(addParticipantOptions);
+                    var Target = new PhoneNumberIdentifier(Participantindentity);
+
+                    var CallInvite = new CallInvite(Target, SourceCallerId);
+                    var addParticipantOptions = new AddParticipantOptions(CallInvite);
+                    await callConnection.AddParticipantAsync(addParticipantOptions);
+                    count++;
+                    logger.LogInformation($"Add Participant: {JsonConvert.SerializeObject(@event)}" + $"participant id :{Participantindentity}");
+                }
+                logger.LogInformation($"List of Participants: {count}" + $"� participant ids :{target}");
```
Comment (moirf) [inline CallAutomation_SimpleIvr/Program.cs:163]: remove the ? marks unrecognized char.

File: CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/appsettings.json (hunk 2 above)
Comment (Rani0608) [reply CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/appsettings.json]: done Irfan.

File: CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/appsettings.json (hunk 3 above)
Comment (Rani0608) [reply CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/appsettings.json]: done.

File: CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder.csproj (hunk 1 above)
Comment (Rani0608) [reply CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder.csproj:16]: No Irfan. I was tested in Local, it is working fine.

File: CallAutomation_SimpleIvr/Program.cs (hunk 5 above)
Comment (Rani0608) [reply CallAutomation_SimpleIvr/Program.cs:163]: I am not changing anything on SimpleIVR ,

**Review by moirf — APPROVED** [review]

**Pull request merged**